icinga2/service.go

```diff
@@ -3,25 +3,31 @@ package icinga2
 import (
 	"fmt"
 	"net/url"
+	"time"
 )
 
 type Service struct {
-	Name               string  `json:"name,omitempty"`
-	DisplayName        string  `json:"display_name"`
-	HostName           string  `json:"host_name"`
-	CheckCommand       string  `json:"check_command"`
-	EnableActiveChecks bool    `json:"enable_active_checks"`
-	Notes              string  `json:"notes"`
-	NotesURL           string  `json:"notes_url"`
-	ActionURL          string  `json:"action_url"`
-	Vars               Vars    `json:"vars"`
-	Zone               string  `json:"zone,omitempty"`
-	CheckInterval      float64 `json:"check_interval"`
-	RetryInterval      float64 `json:"retry_interval"`
-	MaxCheckAttempts   float64 `json:"max_check_attempts"`
-	CheckPeriod        string  `json:"check_period,omitempty"`
-	State              float64 `json:"state,omitempty"`
-	LastStateChange    float64 `json:"last_state_change,omitempty"`
+	Name               string        `json:"name,omitempty"`
+	DisplayName        string        `json:"display_name"`
+	HostName           string        `json:"host_name"`
+	CheckCommand       string        `json:"check_command"`
+	EnableActiveChecks bool          `json:"enable_active_checks"`
+	Notes              string        `json:"notes"`
+	NotesURL           string        `json:"notes_url"`
+	ActionURL          string        `json:"action_url"`
+	Zone               string        `json:"zone,omitempty"`
+	CheckInterval      float64       `json:"check_interval"`
+	RetryInterval      float64       `json:"retry_interval"`
+	MaxCheckAttempts   float64       `json:"max_check_attempts"`
+	CheckPeriod        string        `json:"check_period,omitempty"`
+	State              float64       `json:"state,omitempty"`
+	LastStateChange    float64       `json:"last_state_change,omitempty"`
+	DummyText          string        `json:"dummy_text,omitempty"`
+	DummyState         int           `json:"dummy_state,omitempty"`
+	LabelHeartbeat     bool          `json:"label_heartbeat,omitempty"`
+	BridgeUUID         string        `json:"vars.bridge_uuid"`
+	KeepFor            time.Duration `json:"vars.keep_for"`
+	Templates          []string      `json:"templates,omitempty"`
```

Comment:
Why do we expose a bunch of Signalilo-specific variables in the generic Service type? These fields make no sense for any other application which wants to use this library.

Additionally, I assume that all of these should use the `vars.` prefix, as they're all configured as variables on the Icinga2 service object.